### ISSUE TITLE
In-program editor to make mods?

### ISSUE THREAD

**Issue by thecnoNSMB** [Bambosh/unofficial-homestuck-collection#443]
I have no idea (or expectation) if this is remotely feasible, I just think it might be ideal to have and would make text-editing mods easier to make. I am aware this is incredibly optimistic, and am just making an issue for it so that the idea is written down somewhere.

**Comment by thecnoNSMB** [comment]
#307 is related and would probably have to be done first.

**Comment by thecnoNSMB** [comment]
actually yeah nevermind closing as duplicate of #307
